### DISCUSSION TITLE
Output contact force of floating body and cylindrical  structure

### DISCUSSION THREAD

**Discussion by BiyaoZhai** [DualSPHysics/DualSPHysics#223] [DUALSPHYSICS_V5.2]
Hi, I am doing the simulation of many floating ice blocks impacting on the structure in a wave flume, I was wondering how to output the forces of the floating body on the structure.

**Comment by alexbexe** [comment]
@imestevez can give you some suggestions here

**Comment by imestevez** [comment]
The new relesead version allows to oput both fluid and collision forces using the option:
```
<special>
    <chrono>
        <savedata value="0.01" comment="Saves CSV with data exchange for each time interval (0=all steps)" />
        ...
    </chrono>
    ...
</special>
```
